vllm/entrypoints/openai/rpc/server.py

```diff
@@ -177,11 +174,11 @@ async def run_server_loop(self):
         running_tasks = set()
         while True:
             # Wait for a request.
-            identity, message = await self.socket.recv_multipart()
+            identity, part2, message = await self.socket.recv_multipart()
```

Comment:
For future readers it'd be nice to add a link to some zmq docs here or give this a descriptive name to say what `part2` is. From context here I'm guessing this is routing information for the client-side proxy?

Reply:
Its related to the use of `ROUTER`, will do